unit-testing/index.md

```diff
@@ -1,7 +1,20 @@
 # Module content
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+1. Testing in software development​
```

Comment:
How about adding an overview of all test types? They are specific in react-native and there are not as many materials on this topic as in unit testing

Reply:
Hi!
I am gonna cover it briefly via 5. Testing statistic.
for example:
<img width="539" alt="Screenshot 2022-10-05 at 11 07 11" src="https://user-images.githubusercontent.com/54724990/194011702-5ce99367-4dc5-4ce3-90eb-965529c3c76b.png">

Thanks for your comment :)

Reply:
I would also add tools that can be used for this in React Native, and we need to at least add to the materials how to write other types of tests